### PR TITLE
expanding on median predictor section

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

**Comment by dericed** [comment]
This moves the context-template into the median prediction section. In the RFC version it auto-adds a `Figure #` that use the #anchor to make cross-references to, but it's hard to emulate this behavior exactly in the PDF version. Still I think it makes sense for both the median and context sections to use the same figure and relative-sample codes (tl, l, t, etc). I also defined a median function.

Comments?

File: ffv1.md [hunk 1]
```diff
@@ -111,6 +111,8 @@ log2(a)               the base-two logarithm of a
 
 min(a,b)              the smallest of two values a and b
 
+median(a,b,c)         the numerical middle value in a data set of a, b, and c
```
Comment (michaelni) [inline ffv1.md]: You can define median() as a+b+c-min(a,b,c)-max(a,b,c) if you want a more formal definiton without "middle"

Comment (dericed) [inline ffv1.md]: updated to a mix of the two:

> the numerical middle value in a data set of a, b, and c, i.e. a+b+c-min(a,b,c)-max(a,b,c)

**Comment by dericed** [comment]
PR updated according to @michaelni's feedback. Please re-review.

**Pull request closed without merging**